src/app/_services/auth.service.ts

```diff
@@ -326,13 +342,18 @@ constellationInit = (authConfig:any, tokenInfo:any) => {
       sdkConfigAuth.authService = "pega";
     }
 
+    // Construct path to redirect uri
+    let sRedirectUri=`${window.location.origin}${window.location.pathname}`;
+    const nLastPathSep = sRedirectUri.lastIndexOf("/");
+    sRedirectUri = `${sRedirectUri.substring(0,nLastPathSep+1)}auth.html`
+  
     const authConfig:any = {
       clientId: this.bEmbeddedLogin ? sdkConfigAuth.mashupClientId : sdkConfigAuth.portalClientId,
       authorizeUri: sdkConfigAuth.authorize,
       tokenUri: sdkConfigAuth.token,
       revokeUri: sdkConfigAuth.revoke,
       redirectUri: this.bEmbeddedLogin || this.bUsePopupForRestOfSession || endpoints.loginExperience === loginBoxType.Popup ?
-        `${window.location.origin}/auth.html` : `${window.location.origin}${window.location.pathname}`,
+      sRedirectUri : `${window.location.origin}${window.location.pathname}`,
```

Comment:
Please indent line 356 (as it can be easily mistaken to be another authConfig property names "sRedirectUri" rather than the then clause

Reply:
Sure